src/Models/DataBase/Comment.cs

```diff
@@ -5,6 +5,12 @@ namespace TrainingNet.Models.DataBase
 {
     public class Comment
     {
+        public Comment() { }
+        public Comment(Movie movie, string content)
+        {
+            this.movie = movie;
+            this.content = content;
+        }
         public int Id { get; set; }
         public string content { get; set; }
         public virtual Movie movie { get; set; }
```

Comment:
I didn't see this one, but keep in mind these attributes should begin with a capital letter (no need to change them here, just sth to keep in mind)